### PR TITLE
Fixing issue with tokenType when renewing Id Token

### PR DESCRIPTION
**Pull request by pkanher617**
This PR is to fix part of an issue describe in issue #736 where the tokenType in the response for a acquireTokenSilent call which was renewing an id token was incorrectly displaying the "access_token" type.

### CODE REVIEW

File: lib/msal-core/src/UserAgentApplication.ts [hunk 1]
```diff
@@ -655,6 +655,7 @@ export class UserAgentApplication {
               // App uses idToken to send to api endpoints
               // Default scope is tracked as clientId to store this token
               this.logger.verbose("renewing idToken");
+              this.silentLogin = true;
```
Comment (jasonnutter) [inline lib/msal-core/src/UserAgentApplication.ts]: Can you explain what this option does?

Comment (pkanher617) [reply lib/msal-core/src/UserAgentApplication.ts]: Essentially it sets the fact that we are logging in (requesting only an id token) silently. It isn't a great design, and we are looking at refactoring this to take this out, but for now it exists so I am leveraging it here.

**Comment by pkanher617** [comment]
This is not something we can unit test, which is why I have added a sample.

File: lib/msal-core/src/UserAgentApplication.ts (hunk 1 above)
Comment (sameerag) [reply lib/msal-core/src/UserAgentApplication.ts]: Looks okay but the name bugs me. ATS would have to be ideally silentLogin - but looks like we are adding it for cases of adal idToken -> ATS route only previously. Using the same for an idToken renewal is confusing, Having said that I cannot yet think of a better option. Let's discuss this in person once.

Comment (pkanher617) [reply lib/msal-core/src/UserAgentApplication.ts]: I think this variable doesn't even need to be there, but we can discuss that another time. For now we should approve and merge this.

Comment (sameerag) [reply lib/msal-core/src/UserAgentApplication.ts]: That I agree. Maybe no sense in renaming it now.

**Pull request merged**